compiler/src/test/scala/org/scalajs/core/compiler/test/JSExportTest.scala

```diff
@@ -11,6 +11,8 @@ class JSExportTest extends DirectTest with TestHelpers {
 
   @Test
   def noDoubleUnderscoreExport = {
+    // scalastyle:off
```

Comment:
`scalastyle:off` is quite dangerous. You should probably specify the rule id to turn off here `scalastyle:off underscore.import`